Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -190,17 +187,20 @@ static Color GetAccentColor()
 
 		class AndroidDeviceInfo : DeviceInfo
 		{
-			readonly IDeviceInfoProvider _formsActivity;
+			readonly Context _formsActivity;
 			readonly Size _pixelScreenSize;
 			readonly double _scalingFactor;
 
 			Orientation _previousOrientation = Orientation.Undefined;
 
-			public AndroidDeviceInfo(IDeviceInfoProvider formsActivity)
+			public AndroidDeviceInfo(Context formsActivity)
```

Comment:
If there is no guarantee that it's an `Activity`, then `_formsActivity` should probably be renamed to `context`. Right now, it implies that it's an `Activity`.